Header header
-------
geometry_msgs/Pose[] original_path
geometry_msgs/Pose[] predictive_path
